forum/forum/tests_setup.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework.test import APITestCase
+from rest_framework_simplejwt.tokens import AccessToken
+
+from users.models import CustomUser
+
+
+class TestUserSetupMixin(APITestCase):
+    def setup_test_user(self) -> None:
+        new_user = CustomUser.objects.create(
+            first_name="test_first",
+            last_name="test_last",
+            email="test@gmail.com"
+        )
```

Comment:
Ensure that a password is set for the user so that it can be used for login if needed.
_Suggestions_
```
    def setUp(self) -> None:
        super().setUp()
        self.setup_test_user()

    def setup_test_user(self) -> None:
        self.test_user = CustomUser.objects.create_user(
            first_name="test_first",
            last_name="test_last",
            email="test@gmail.com",
            password="test_password"
        )
```